components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -47,6 +78,24 @@ func newStatusCmd() *cobra.Command {
 
 	return statusCmd
 }
+func newStatusSummaryCmd() *cobra.Command {
+	var statusSummaryCmdFlags = StatusSummaryCmdFlags{}
+	var statusSummaryCmd = &cobra.Command{
+		Use:   "summary",
+		Short: "Retrieve Chef Automate status-summary",
+		Long:  "Retrieve Chef Automate status node summary for HA deployment",
+		RunE:  runStatusSummaryCmdFunc(&statusSummaryCmdFlags),
+	}
+	statusSummaryCmd.PersistentFlags().StringVarP(&statusSummaryCmdFlags.automateIp, "automate-ips", "A", "", "Get automate Status by ip addresses")
```

Comment:
@shaik80  these flags don't seem consistent to cert rotate commands

Reply:
I have taken reference from had add node, in cert rotate commands its -a flag

Reply:
made changes